python/pyspark/sql/pandas/types.py

```diff
@@ -124,12 +176,40 @@ def to_arrow_type(dt: DataType) -> "pa.DataType":
     return arrow_type
 
 
-def to_arrow_schema(schema: StructType) -> "pa.Schema":
-    """Convert a schema from Spark to Arrow"""
+def to_arrow_schema(
+    schema: StructType,
+    error_on_duplicated_field_names_in_struct: bool = False,
+    timestamp_utc: bool = True,
```

Comment:
Seems like we don't use this? `timestamp_utc`

Reply:
It is used in one place in the tests. See `test_createDataFrame_arrow_nested_timestamp`.